src/components/Checkout/DonationItem.vue

```diff
@@ -663,6 +666,93 @@
 				</div>
 			</div>
 		</template>
+		<template v-if="donateItemExperimentVersion === 'e' && orderTotalVariant && hasLoans">
+			<div class="tw-flex tw-flex-col tw-w-full tw-pb-4">
+				<div class="tw-flex tw-flex-row tw-w-full">
+					<!-- donation text -->
+					<div class="tw-w-auto tw-text-left md:tw-text-right tw-flex-1">
+						<h2
+							class="tw-text-h3"
+							data-testid="basket-donation-title"
+						>
+							Donate to Kiva:
+						</h2>
+					</div>
+
+					<!-- donation total -->
+					<div
+						class="tw-flex-none tw-w-auto"
+					>
+						<div
+							v-show="!editDonation"
+							class="tw-block tw-text-right tw-ml-1"
+						>
+							<button
+								class="donation-amount tw-text-h3"
+								data-testid="basket-donation-edit-button-combined"
+								v-kv-track-event="['basket', 'Edit Donation']"
+								@click="enterEditDonation"
+								title="Edit Donation"
+							>
+								{{ formattedAmount }}
+								<kv-material-icon
+									role="img"
+									aria-label="Edit Donation"
+									title="Edit Donation"
+									class="edit-donation
+											tw-text-action
+											tw-w-2.5
+											tw-ml-2
+											tw-h-3.5
+											tw-py-0.5
+											tw-align-bottom
+									"
+									name="pencil"
+									:icon="mdiPencil"
+								/>
+							</button>
+						</div>
+						<div v-show="editDonation" class="small-12 columns donation-amount-input-wrapper">
+							<kv-text-input
+								class="donation-amount-input"
+								data-testid="basket-donation-edit-input"
+								name="donation"
+								id="donation"
+								v-model="amount"
+								@blur="validateInput"
+								@keyup.enter.prevent="updateDonation()"
+							/>
+							<kv-button
+								variant="secondary"
+								class="update-donation-inline-button"
+								data-testid="basket-donation-edit-submit"
+								@click="updateDonation()"
+							>
+								Update
+							</kv-button>
+							<button
+								class="show-for-medium remove-wrapper"
```

Comment:
I'm guessing `show-for-medium` is foundation as well. Since these classes are already in use in this component, no need to replace, but we'll need to replace them at some point.